src/Mapping.cc

```diff
@@ -114,8 +114,11 @@ void Mapping::_tran(ConstArray2D const &from, bool doForward, Array2D const &to)
     detail::assertEqual(to.getSize<0>(), "to.size[0]", static_cast<std::size_t>(nToAxes), "to coords");
     detail::assertEqual(from.getSize<1>(), "from.size[1]", to.getSize<1>(), "to.size[1]");
     int const nPts = from.getSize<1>();
-    astTranN(getRawPtr(), nPts, nFromAxes, nPts, from.getData(), static_cast<int>(doForward), nToAxes, nPts,
-             to.getData());
+    // astTranN treats 0 points as an error and the call isn't needed anyway
```

Comment:
You might want to explicitly say that the caller is responsible for making `to` be an empty array of the correct dimensions; it took me a while to convince myself that this method really didn't need to do anything.

Reply:
The caller is always responsible for allocating "to", as per the main description: "Implement applyForward and applyInverse, putting the results into a pre-allocated 2-D array." However, I have added "the array must be pre-allocated with the correct dimensions." to the description of the "to" argument.